CefSharp.BrowserSubprocess.Core/JavascriptObjectWrapper.cpp

```diff
@@ -14,6 +14,22 @@ namespace CefSharp
 {
     void JavascriptObjectWrapper::Bind()
     {
+        auto objectName = StringUtils::ToNative(_object->JavascriptName);
+
+        if (_object->IsNull)
+        {
+            auto javascriptObject = CefV8Value::CreateNull();
+            if (V8Value.get() && !V8Value->HasValue(objectName))
+            {
+                V8Value->SetValue(objectName, javascriptObject, V8_PROPERTY_ATTRIBUTE_NONE);
+            }
+            else
+            {
+                V8Value = javascriptObject;
```

Comment:
What scenario is this condition met? Setting the `V8Value` to a null version seems strange. Code block in general needs some descriptive comments about what it's purpose is.